sensor_msgs/JointState goal_joint
---
int8 success